ApiDoctor.Validation.UnitTests/SchemaBuilderTests.cs

```diff
@@ -37,7 +37,7 @@ public class SchemaBuilderTests
 
         public static JsonSchema SimpleSchemaExample()
         {
-            var obj = new { stringProp = "string", urlProp = "url", numberProp = 123, dateProp = "datetime", floatProp = 123.121 };
+            var obj = new { stringProp = "string", urlProp = "url", numberProp = 123, dateProp = "datetime", floatProp = 123.121d };
```

Comment:
is the trailing `d` necessary? i'd prefer if the tests reflected what our data usually looks like

Reply:
Fixed.